styles/styles.css

```diff
@@ -58,6 +58,15 @@ header {
     top: 10px;
 }
 .exit { right: 20px; }
+#level1 {
+  left: 150px;
+}
+#level2 {
+  left: 225px;
+}
+#level3 {
+  left: 300px;
+}
```

Comment:
@OliviaJorasch do you think you can try to add some Responsive designs in here for mobile? You can try it out with the code inspector in chrome and emulate your phone when you develop.

Check this resource on @media css properties: https://developers.google.com/web/fundamentals/design-and-ui/responsive/#css-media-queries